### PR TITLE
C2c addons label direct print

### PR DESCRIPTION
**Pull request by damdam-s**

### CODE REVIEW

**Comment by guewen** [comment]
thanks :+1:

**Comment by mdietrichc2c** [comment]
:+1:

File: delivery_label_direct_print/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Guewen Baconnier
+#    Copyright 2014 Camptocamp SA
```
Comment (yvaucher) [inline delivery_label_direct_print/__openerp__.py:5]: Shouldn't it be 2015 here as in other files ?

**Comment by yvaucher** [comment]
Just a question, otherwise :+1:

File: delivery_label_direct_print/__openerp__.py (hunk 1 above)
Comment (damdam-s) [inline delivery_label_direct_print/__openerp__.py:5]: Yes of course.
But actually, the original module has just been migrated as this so I didn't change dates.
Should I do it ?

Comment (yvaucher) [inline delivery_label_direct_print/__openerp__.py:5]: Let's ask @guewen if this is a typo ?

Comment (damdam-s) [inline delivery_label_direct_print/__openerp__.py:5]: @guewen what about @yvaucher previous comment ?

**Comment by damdam-s** [comment]
Peut-on merger cette PR ?

**Comment by yvaucher** [comment]
Okay demandé à @guewen en direct, je merge

**Pull request merged**